src/doing-business-with-us/_vars-doing-business-with-us.html

```diff
@@ -7,3 +7,4 @@
       (path + 'small-businesses/', 'small-businesses', 'Small, Women-Owned, and Minority-Owned Businesses')
     ])
 ] -%}
+{% set breadcrumb_items = [(path, path, 'Doing Business with us')] %}
```

Comment:
Could the `Doing Business with us` and the `nav_items` index text share a variable?